<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
 
 background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, radius: 1.35, stop: 0 #fff, stop: 1 #888) ;/*background: linear-gradient(rgba(0,0,0,.5), rgba(0,0,0,.5)),url(feuille.jpg); 
    background-position: center;
    background-size: cover;
    display: flex;
    justify-content: center;
    align-items: center;
    font-family: 'Poppins', sans-serif*/
 
 
}



QPushButton {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background-color:pink
}

 QPushButton:hover:!pressed{ background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF); } 
QPushButton:hover { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF); }
/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:white
}*/
</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Produits</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Nom produit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Stock</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_viedo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_stock">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="valider">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>210</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>991</width>
        <height>631</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/lovepik-financial-real-estate-black-gold-background-image_400084852.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>fk fournisseur</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>fk</string>
       </property>
      </item>
     </widget>
     <zorder>label_14</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_id</zorder>
     <zorder>lineEdit_nom</zorder>
     <zorder>lineEdit_viedo</zorder>
     <zorder>lineEdit_stock</zorder>
     <zorder>valider</zorder>
     <zorder>lineEdit_prix</zorder>
     <zorder>label_52</zorder>
     <zorder>label_6</zorder>
     <zorder>comboBox_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Read</string>
     </attribute>
     <widget class="QTableView" name="tab_produit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1011</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chercher_id">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>460</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:white
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>230</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="update_video">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>290</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:rgb(34,36,44);
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}*/</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="update_prix">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>380</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:rgb(34,36,44);
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}*/</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>330</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>stock</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="update_nom">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:rgb(34,36,44);
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>380</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>260</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="update_stock">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>320</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:rgb(34,36,44);
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>300</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>video</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>-320</x>
        <y>-30</y>
        <width>1331</width>
        <height>701</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/lovepik-financial-real-estate-black-gold-background-image_400084852.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="read">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_del">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:rgb(34,36,44);
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="update_id">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:rgb(34,36,44);
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>220</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="update_fk">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>430</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QLineEdit{
border: 2px solid rgb(37,39,48);
border -radius: 20px;
color: #FFF;
padding -left: 20px;
padding -right: 20px;
background-color:rgb(34,36,44);
}
QLineEdit:hover{
border: 2px solid rgb(48,50,62);
}
QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color: rgb(43,45,56);
}*/</string>
      </property>
     </widget>
     <zorder>label_13</zorder>
     <zorder>tab_produit</zorder>
     <zorder>rechercher_2</zorder>
     <zorder>chercher_id</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>update_video</zorder>
     <zorder>update_prix</zorder>
     <zorder>label_12</zorder>
     <zorder>update_nom</zorder>
     <zorder>label_20</zorder>
     <zorder>label_10</zorder>
     <zorder>update_stock</zorder>
     <zorder>label_9</zorder>
     <zorder>read</zorder>
     <zorder>pushButton_delete</zorder>
     <zorder>lineEdit_del</zorder>
     <zorder>update_id</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>comboBox</zorder>
     <zorder>update_fk</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
